### PR TITLE
Implement LessonCreator

### PR DESCRIPTION
**Pull request by djrenren**
Implements the LessonCreator with word creation inside of it. This change is client-side only

### CODE REVIEW

File: client/src/components/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {RaisedButton} from 'material-ui';
+import Page from './util/Page';
+import * as React from 'react';
+import { browserHistory } from 'react-router';
+
+
+export const Home = (_: any) => {
+    return (<Page>
+        <RaisedButton primary={true} label='New Lesson'
```
Comment (7imbrook) [inline client/src/components/Home.tsx:9]: Why doesn't Link work?

**Review by 7imbrook — APPROVED** [review]
I'm assuming that the lesson will be fully backed (and only backed) by a reducer in the future? Everything should be in a container for this for when we start persisting state

File: client/src/components/Home.tsx (hunk 1 above)
Comment (djrenren) [reply client/src/components/Home.tsx:9]: Cuz when we hookup to backend it'll need to be more complex and I wanted a button

**Comment by djrenren** [comment]
LessonCreator and WordCreator underneath it, act like any other input element under react so yeah throwing a reducer under them won't be hard. Unfortunately, the best design I see requires an id for a new lesson before opening the new lesson page. So I was avoiding the action/reducer flow at the moment

**Pull request merged**